c25k/index.html

```diff
@@ -47,9 +47,9 @@ <h2>Who is Couch to 5K for?</h2>
       <h2>When does it start?</h2>
       <p>There will be three Couch to 5K groups in 2025.</p>
```

Comment:
I think @ear1grey is suggeesting that the "groups" in this sentence should also be "cohorts"